CombinatorialGames/Counterexamples/Multiplication.lean

```diff
@@ -18,61 +18,34 @@ is colloquially known as `star`, so we use the name `star'` for the second. We p
 `star ≈ star'` and `star * star ≈ star`, but `star' * star ≉ star`.
 -/
 
-namespace PGame
+noncomputable section
+open IGame
 
-open PGame
+namespace IGame
 
 /-- The game `{-1, 0 | 0, 1}`, which is equivalent but not identical to `*`. -/
-def star' : PGame := ofLists [0, -1] [0, 1]
+def star' : IGame := {{0, -1} | {0, 1}}ᴵ
+
+@[simp] theorem leftMoves_star' : leftMoves star' = {0, -1} := leftMoves_ofSets ..
+@[simp] theorem rightMoves_star' : rightMoves star' = {0, 1} := rightMoves_ofSets ..
 
 /-- `*'` is its own negative. -/
-theorem neg_star' : -star' = star' := by
-  simp [star']
+theorem neg_star' : -star' = star' := by simp [star']
 
 /-- `*'` is equivalent to `*`. -/
-theorem star'_equiv_star : star' ≈ star := by
-  have le : star' ≤ star := by
-    apply PGame.le_of_forall_lf
-    · rintro ⟨i⟩
-      fin_cases i
-      · exact zero_lf_star
-      · exact (neg_lt_zero_iff.2 PGame.zero_lt_one).trans_lf zero_lf_star
-    · exact fun _ ↦ lf_zero_le.2 ⟨⟨0, Nat.zero_lt_two⟩, le_rfl⟩
-  constructor
-  case' right => dsimp; rw [← neg_le_neg_iff, neg_star, neg_star']
-  assumption'
+theorem star'_equiv_star : star' ≈ ⋆ := by game_cmp
 
 /-- The equation `** = *` is an identity, though not a relabelling. -/
```

Comment:
Whoops, forgot to mention that "relabelling" isn't really a concept we're using anymore. In fact, even though it's not really relevant to the counterexample, it might be worth proving that `** = *` equality.